modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/query/ElasticsearchQueryTranslatorTest.java

```diff
@@ -104,6 +108,32 @@ public void testTranslateInnerBoostBooleanQuery() {
 			String.valueOf(innerQueryBuilder.boost()));
 	}
 
+	@Test
+	public void testTranslateQueryWithMoreTermsThanTheMaximumAllowed() {
+		TermsQuery termsQuery = new TermsQueryImpl("groupId");
+
+		TermsQueryTranslator termsQueryTranslator =
+			new TermsQueryTranslatorImpl();
+
+		ReflectionTestUtil.setFieldValue(
+			_elasticsearchQueryTranslator, "_termsQueryTranslator",
+			termsQueryTranslator);
+
+		termsQuery.addValues("0", "1", "2", "3", "4", "5", "6", "7", "8", "9");
```

Comment:
maybe randomize values here

Reply:
I don't think this is a good idea, as it would add more complexity to the test execution without interfering with the test itself. The values ​​added to the term query are not important for the use case that the test is addressing, what matters is the number of values ​​added.